epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py

```diff
@@ -716,9 +721,13 @@ def write_hardcoded_parameters(self, params):
             res = '// (none)\n'
             return res
         pardef = pardef.replace('std::complex<','cxsmpl<') # custom simplex complex class (with constexpr arithmetics)
-        parset = parset.replace('std::complex<','cxsmpl<') # custom simplex complex class (with constexpr arithmetics)
+        parset = parset.replace('std::complex<','cxsmpl<') # NB this is really needed twice! (if there are two std::complex on the same line)
```

Comment:
Are you sure?
https://www.w3schools.com/python/ref_string_replace.asp
said that "ALL occurrences" will be replaced. Or am I wrong that parset is of type string?